devicon.json

```
@@ -7557,6 +7557,30 @@
         "color": "#5c4ee5",
         "aliases": []
     },
+    {
+        "name": "tex",
+        "altnames": [],
+        "tags": [
+            "typesetting-system",
+            "markup",
+            "tex"
+        ],
+        "versions": {
+            "svg": [
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain-wordmark"
+            ]
+        },
+        "color": "#000000",
+        "aliases":[
+            {
+                "base": "plain-wordmark",
+                "alias": "plain"
+            }
+        ]
```

Same here as with the [YAML](https://github.com/devicons/devicon/pull/1448) PR, the icon should be changed to `original`, and the aliases should be changed as follows:
```suggestion
        "versions": {
            "svg": [
                "original"
            ],
            "font": [
                "original"
            ]
        },
        "color": "#000000",
        "aliases": [
            {
                "base": "original",
                "alias": "original-wordmark"
            },
            {
                "base": "original",
                "alias": "plain"
            },
            {
                "base": "original",
                "alias": "plain-wordmark"
            }
        ]
```